chronos/models/users.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+const { db } = require('./db');
+
+export const getUserWhoIsNotContributorByGithubProviderId = async (
+  githubUserId: string
+) => {
+  return await db
+    .table('users')
+    .getAll(githubUserId, { index: 'githubProviderId' })
```

Comment:
I guess you can reuse the constant I mentioned for `githubProviderId` in `api/models/user.js` :)